feature-libs/user/account/components/login-form-cdc/login-form-cdc.component.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { filter, take } from 'rxjs';
+import { OAuthService } from 'angular-oauth2-oidc';
+import { createAuthConfig } from './auth.config';
+import { AuthService, BaseSiteService } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-login-form-cdc',
+  templateUrl: './login-form-cdc.component.html',
+})
+export class LoginFormCDCComponent implements OnInit {
+  protected baseSite: string = '';
+  constructor(
+    protected oauthService: OAuthService,
+    protected baseSiteService: BaseSiteService,
+    protected auth: AuthService
+  ) {}
+
+  ngOnInit(): void {
+    this.baseSiteService
+      .getActive()
+      .pipe(take(1))
+      .subscribe((data) => (this.baseSite = data));
```

Comment:
See my comments in below, if you call this.baseSiteService.get() , you don't need this.baseSite at all?

Reply:
need to use ` this.baseSite` to construct redirect url when calling createAuthConfig()